docs/CLI.md

```diff
@@ -61,3 +61,12 @@ gulp has very few flags to know about. All other flags are for tasks to use if n
 
 **--log-level**, **-L**
     Set the loglevel. -L for least verbose and -LLLL for most verbose. -LLL is default.
+
+**--config**, **-c**
+    Manually set path of configuration file.
+
+**--ignore-config**
```

Comment:
Certainly, we don't need it right away. But in the future, when a lot of configurations can be customized by the config files, there would be some cases that an user want to off his own configurations, e.g. for testing configuration files temporarily, for trying or debugging other person's code not to be affected by them, etc.